### PR TITLE
Fix empy

### PR DESCRIPTION
**Pull request by pablogs9**
FiSigned-off-by: Pablo Garrido <pablogs9@gmail.com>

### CODE REVIEW

File: extra_script.py [hunk 1]
```diff
@@ -83,7 +83,7 @@ def build_microros(*args, **kwargs):
     ##############################
 
     pip_packages = [x.split("==")[0] for x in os.popen('{} -m pip freeze'.format(env['PYTHONEXE'])).read().split('\n')]
-    required_packages = ["catkin-pkg", "lark-parser", "empy", "colcon-common-extensions", "importlib-resources", "pyyaml", "pytz", "markupsafe==2.0.1"]
+    required_packages = ["catkin-pkg", "lark-parser", "empy", "colcon-common-extensions", "importlib-resources", "pyyaml", "pytz", "markupsafe==2.0.1", "empy==3.3.4ss"]
```
Comment (pablogs9) [inline extra_script.py:86]:
```suggestion
    required_packages = ["catkin-pkg", "lark-parser", "empy", "colcon-common-extensions", "importlib-resources", "pyyaml", "pytz", "markupsafe==2.0.1", "empy==3.3.4"]
```

**Comment by gabryelreyes** [comment]
Hi @pablogs9, is it intentional that empy is written in the dependency list in the 3rd place without a specific version, and then at the end with the 3.3.4 version? Shouldn't it only be defined with the 3.3.4 version and remove the other one?

**Comment by pablogs9** [comment]
Nope, just an error. I`ll open a fix

**Comment by pablogs9** [comment]
Done: https://github.com/micro-ROS/micro_ros_platformio/pull/157

**Pull request merged**